ntp-proto/src/packet/mod.rs

```diff
@@ -1070,12 +1180,16 @@ mod tests {
         assert!(NtpPacket::deserialize(packet, &NoCipher).is_err());
         let packet = b"\x14\x02\x06\xe9\x00\x00\x02\x36\x00\x00\x03\xb7\xc0\x35\x67\x6c\xe5\xf6\x61\xfd\x6f\x16\x5f\x03\xe5\xf6\x63\xa8\x76\x19\xef\x40\xe5\xf6\x63\xa8\x79\x8c\x65\x81\xe5\xf6\x63\xa8\x79\x8e\xae\x2b";
         assert!(NtpPacket::deserialize(packet, &NoCipher).is_err());
-        let packet = b"\x2B\x02\x06\xe9\x00\x00\x02\x36\x00\x00\x03\xb7\xc0\x35\x67\x6c\xe5\xf6\x61\xfd\x6f\x16\x5f\x03\xe5\xf6\x63\xa8\x76\x19\xef\x40\xe5\xf6\x63\xa8\x79\x8c\x65\x81\xe5\xf6\x63\xa8\x79\x8e\xae\x2b";
-        assert!(NtpPacket::deserialize(packet, &NoCipher).is_err());
         let packet = b"\x34\x02\x06\xe9\x00\x00\x02\x36\x00\x00\x03\xb7\xc0\x35\x67\x6c\xe5\xf6\x61\xfd\x6f\x16\x5f\x03\xe5\xf6\x63\xa8\x76\x19\xef\x40\xe5\xf6\x63\xa8\x79\x8c\x65\x81\xe5\xf6\x63\xa8\x79\x8e\xae\x2b";
         assert!(NtpPacket::deserialize(packet, &NoCipher).is_err());
         let packet = b"\x3B\x02\x06\xe9\x00\x00\x02\x36\x00\x00\x03\xb7\xc0\x35\x67\x6c\xe5\xf6\x61\xfd\x6f\x16\x5f\x03\xe5\xf6\x63\xa8\x76\x19\xef\x40\xe5\xf6\x63\xa8\x79\x8c\x65\x81\xe5\xf6\x63\xa8\x79\x8e\xae\x2b";
         assert!(NtpPacket::deserialize(packet, &NoCipher).is_err());
+
+        #[cfg(not(feature = "ntpv5"))]
+        {
+            let packet = b"\x2C\x02\x06\xe9\x00\x00\x02\x36\x00\x00\x03\xb7\xc0\x35\x67\x6c\xe5\xf6\x61\xfd\x6f\x16\x5f\x03\xe5\xf6\x63\xa8\x76\x19\xef\x40\xe5\xf6\x63\xa8\x79\x8c\x65\x81\xe5\xf6\x63\xa8\x79\x8e\xae\x2b";
+            assert!(NtpPacket::deserialize(packet, &NoCipher).is_err());
+        }
```

Comment:
why does ntpv5 change this test?

Reply:
The test actually checks that a packet with NTP version 5 does produce an error. I guess we could instead of disabling it just test that it returns `ok` with the `ntpv5` feature.

Reply:
I added a comment to why it is disables. It can not test for `Ok()` with the feature because it is not a valid V5 packet... So I guess this is the easiest way forward, as all other tests only check for `.is_err()` and succesful parsing is tested at another point.